test/key_test.py

```diff
@@ -45,6 +47,17 @@ def test_name(self):
         self.assertEqual(pygame.key.name(pygame.K_0), "0")
         self.assertEqual(pygame.key.name(pygame.K_SPACE), "space")
 
+    def test_key_code(self):
```

Comment:
You should add first 4 asserts under `if not SDL1`, right now they are checked in every case (and as far as I have understood key_code should be undefined with SDL1

Reply:
Good catch! I've fenced these off now... And now I see I could have just taken your version from teis conversation thread.

You can tell I'm not much of a GitHub master.